src/bindings/xfconf/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+include (FindXfconf)
+
+if (NOT XFCONF_FOUND)
+	exclude_binding (xfconf "xfconf >= 4.16 required for xfconf backend")
+	return ()
+endif ()
+
+add_binding (xfconf)
+
+add_headers (ELEKTRA_HEADERS)
+
+include_directories (${GLIB_INCLUDE_DIRS})
+include_directories (${XFCONF_INCLUDE_DIRS})
+
+file (GLOB XFCONF_BINDING_HEADERS *.h)
+file (GLOB XFCONF_BINDING_SOURCES *.c)
+
+set (SOURCES ${XFCONF_BINDING_HEADERS} ${XFCONF_BINDING_SOURCES} ${ELEKTRA_HEADERS})
+add_library (xfconfbinding SHARED ${SOURCES})
```

Comment:
Please also install the library in its own component (e.g. see src/bindings/swig/lua and scripts/cmake/ElektraPackaging.cmake).

Reply:
I will have a look on that.

Reply:
Should be done with 4376d1a09a80212e4768477d032a2b1b68595e19.